rockcraft.yaml

```
@@ -25,28 +25,41 @@ parts:
       - llvm
     prime:
       - usr/local/lib/x86_64-linux-gnu/*
+  
+  libbpf:
+    plugin: nil
+    source: https://github.com/libbpf/libbpf.git
+    source-tag: v0.5.0
+    override-build: |
+      cd src/
+      make install
 
   cndp:
     after:
       - xdp
+      - libbpf
     plugin: meson
     source: https://github.com/CloudNativeDataPlane/cndp.git
     source-commit: d5ce4b9edc2e7ddb46a61b395deffafaf11a0500
```

```suggestion
    source-tag: v24.07.0
```

We can use latest available tag.

---

ok, done